packages/web-components/src/components/locale-modal/locale-search.ts

```diff
@@ -217,14 +222,17 @@ class C4DLocaleSearch extends ThrottedInputMixin(
             size="${INPUT_SIZE.LARGE}"
             data-autoid="${c4dPrefix}--locale-modal__filter">
           </cds-search>
-          <div class="${prefix}--visually-hidden" aria-live="polite"></div>
-          <p class="${c4dPrefix}--locale-modal__search-text">
+          <div
+            class="${prefix}--visually-hidden"
+            aria-live="polite"
+            part="polite"></div>
```

Comment:
`polite` refers to how [`aria-live` updates](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live) behave for users of assistive tech, it doesn't really describe this element. I want to suggest something like `aria-updates`
```suggestion
            part="aria-updates"></div>
```